executors/ovhapi/executor.go

```diff
@@ -110,16 +112,38 @@ func (Executor) Run(testCaseContext venom.TestCaseContext, l venom.Logger, step
 		return nil, err
 	}
 
+	if insecure, err := ctx.GetBool("insecureTLS"); err == nil && insecure {
+		client.Client.Transport = &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+		}
+	}
+
 	// get request body from file or from field
 	requestBody, err := e.getRequestBody()
 	if err != nil {
 		return nil, err
 	}
 
+	req, err := client.NewRequest(e.Method, e.Path, requestBody, !e.NoAuth)
+	if err != nil {
+		return nil, err
+	}
+
+	if e.Headers != nil {
+		for key := range e.Headers {
+			req.Header.Add(key, e.Headers[key])
+		}
+	}
+
 	// do api call
-	resp := new(interface{})
```

Comment:
Why did you renamed it ? Idiomatic naming is `resp`, that is what the standard library is using in packahe `net/http`.